tools/wptrunner/wptrunner/executors/protocol.py

```diff
@@ -28,6 +28,18 @@ def __init__(self, executor, browser):
             assert not hasattr(self, name)
             setattr(self, name, cls(self))
 
+    def update(self, target, source):
```

Comment:
I would have just made this a free function called `merge_dicts` or something.

Reply:
updated. I am not exactly sure what is "a free function". Is the new patch what you expected?